srcpkgs/komorebi/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'komorebi'
+pkgname=komorebi
+version=2.2.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config"
+makedepends="vala-devel gtk+3-devel libgee08-devel clutter-gtk-devel clutter-devel webkit2gtk-devel clutter-gst3-devel glib-devel"
+short_desc="Beautiful and customizable wallpapers manager for Linux"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Komorebi-Fork/komorebi"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=4074751825cf031cca63c647372494446dab2492e8311ff439785527b3518c5b
+
+if [ "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" vala glib-devel"
+fi
```

Comment:
This should be in hostmakedeps unconditionally, the fact that it was available for native builds is an accident in that `vala-devel` depends on `vala`, not by design. Same for `glib-devel`.

Reply:
Ah, I see. Thanks for the heads up! Added to `hostmakedepends` unconditionally!